src/fides/data/privacy_notices/privacy_notice_templates.yml

```
@@ -1,4 +1,400 @@
+        not_available: 0
+        opt_out: 1
+        not_opt_out: 2
+    - region: us_ct
+      notice:
+      mechanism:
+      - field: KnownChildSensitiveDataConsents
+        not_available: 000
+        opt_out: 111
+        not_opt_out: 222
+    - region: us_ut
+      notice:
+      mechanism:
+      - field: KnownChildSensitiveDataConsents
+        not_available: 0
+        opt_out: 1
+        not_opt_out: 2
+    - region: us_va
+      notice:
+      mechanism:
+      - field: KnownChildSensitiveDataConsents
+        not_available: 0
+        opt_out: 1
+        not_opt_out: 2
+
+
+
+### GPP US National ###
+- name: Sales, Sharing, Targeted Advertising
+  notice_key: sales_sharing_targeted_advertising_gpp_us_national
+  id: pri_gpp_us_nat_1 # TODO: update
+  description: Sales opt out notice & mechanism; Sharing opt out notice & mechanism; Targeted advertising opt out notice & mechanism; Sharing notice only # TODO: this looks wrong
+  internal_description: Data sales and sharing (GPP, California) # TODO: update?
+  regions:
+    - us
+  consent_mechanism: opt_out
+  data_uses:
+    - marketing.advertising.first_party.targeted
+    - marketing.advertising.third_party.targeted
+  enforcement_level: not_applicable
+  disabled: True
+  has_gpc_flag: False # TODO: not sure?
+  displayed_in_privacy_center: True
+  displayed_in_overlay: True
+  displayed_in_api: False
+  framework: gpp_us_national
+  gpp_field_mapping:
+    - region: us
+      notice:
+        - SaleOptOutNotice
+        - SharingOptOutNotice
+        - SharingNotice
+      mechanism:
+        - field: SaleOptOut
+          not_available: 0
+          opt_out: 1
+          not_opt_out: 2
+        - field: SharingOptOut
+          not_available: 0
+          opt_out: 1
+          not_opt_out: 2
+
+- name: Sensitive Personal Data Sharing
+  notice_key: sensitive_personal_data_sharing_gpp_us_national
+  id: pri_gpp_us_nat_2 # TODO: update
+  description: We may transfer or share Sensitive Personal Data about you with third parties for the purpose of [performance, analytics and reporting]. You can learn more about what information is used for this purpose in our privacy notice.
+  internal_description: Data sales and sharing (GPP, California) # TODO: update?
+  regions:
+    - us
+  consent_mechanism: notice_only
+  # data_uses: data uses must be applied manually
+  enforcement_level: not_applicable
+  disabled: True
+  has_gpc_flag: False # TODO: not sure?
+  displayed_in_privacy_center: True
+  displayed_in_overlay: True
+  displayed_in_api: False
+  framework: gpp_us_national
+  gpp_field_mapping:
+    - region: us
+      notice:
+        - SensitiveDataLimitUseNotice
+        - SensitiveDataProcessingOptOutNotice
+      mechanism:
+        - field: SensitiveDataProcessing
+          not_available: 000000000000
+          opt_out: 111111111111
+          not_opt_out: 222222222222
+
+- name: Known Child Sensitive Data Consents (GPP, US National)
```

only this one has the GPP parens in the name?

```suggestion
- name: Known Child Sensitive Data Consents
```

---

yup, just a miss in my cleanup i think. thanks for catching!